package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "threeflows-v2",
+  "version": "0.0.1-prealpha.0",
+  "description": "Redesign of TSL threeflows app",
```

Comment:
copy from the current one or try to write a better one actually describing the app?